tests/src/test/scala/system/basic/WskBasicPythonTests.scala

```diff
@@ -1,142 +0,0 @@
-    val name = "basicInvoke"
-    assetHelper.withCleaner(wsk.action, name) { (action, _) =>
-      action.create(name, Some(TestUtils.getTestActionFilename("hello.py")))
-    }
-
-    withActivation(wsk.activation, wsk.action.invoke(name, Map("name" -> "Prince".toJson))) {
-      _.response.result.get.toString should include("Prince")
-    }
-  }
-
-  it should "invoke an action with a non-default entry point" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
-    val name = "nonDefaultEntryPoint"
-    assetHelper.withCleaner(wsk.action, name) { (action, _) =>
-      action.create(name, Some(TestUtils.getTestActionFilename("niam.py")), main = Some("niam"))
-    }
-
-    withActivation(wsk.activation, wsk.action.invoke(name, Map())) {
-      _.response.result.get.fields.get("greetings") should be(Some(JsString("Hello from a non-standard entrypoint.")))
-    }
-  }
-
-  it should "invoke an action from a zip file with a non-default entry point" in withAssetCleaner(wskprops) {
-    (wp, assetHelper) =>
-      val name = "pythonZipWithNonDefaultEntryPoint"
-      assetHelper.withCleaner(wsk.action, name) { (action, _) =>
-        action.create(
-          name,
-          Some(TestUtils.getTestActionFilename("python.zip")),
-          main = Some("niam"),
-          kind = Some("python:default"))
-      }
-
-      withActivation(wsk.activation, wsk.action.invoke(name, Map("name" -> "Prince".toJson))) {
-        _.response.result.get shouldBe JsObject("greeting" -> JsString("Hello Prince!"))
-      }
-  }
-
-  Seq("python:2", "python:3").foreach { kind =>
-    it should s"invoke a $kind action" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
-      val name = s"${kind.replace(":", "")}-action"
-      assetHelper.withCleaner(wsk.action, name) { (action, _) =>
-        action.create(name, Some(TestUtils.getTestActionFilename("pythonVersion.py")), kind = Some(kind))
-      }
-
-      withActivation(wsk.activation, wsk.action.invoke(name)) {
-        _.response.result.get shouldBe JsObject("version" -> JsNumber(kind.takeRight(1).toInt))
-      }
-    }
-  }
-
-  it should "invoke an action and confirm expected environment is defined" in withAssetCleaner(wskprops) {
-    (wp, assetHelper) =>
-      val name = "stdenv"
-      assetHelper.withCleaner(wsk.action, name) { (action, _) =>
-        action.create(name, Some(TestUtils.getTestActionFilename("stdenv.py")))
-      }
-
-      withActivation(wsk.activation, wsk.action.invoke(name)) { activation =>
-        val result = activation.response.result.get
-        result.fields.get("error") shouldBe empty
-        result.fields.get("auth") shouldBe Some(
-          JsString(WhiskProperties.readAuthKey(WhiskProperties.getAuthFileForTesting)))
-        result.fields.get("edge").toString.trim should not be empty
-      }
-  }
-
-  it should "invoke an invalid action and get error back" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
```

Comment:
duplicate - but unit test doesn't check error message as done here, should tighten unit test accordingly.